Entities/MOPixel.cpp

```diff
@@ -247,13 +248,18 @@ namespace RTE {
 				break;
 		}
 
-		acquire_bitmap(targetBitmap);
-		putpixel(targetBitmap, m_Pos.GetFloorIntX() - targetPos.m_X, m_Pos.GetFloorIntY() - targetPos.m_Y, drawColor);
-		release_bitmap(targetBitmap);
+#ifndef DRAW_MOID_LAYER
+		if (mode != g_DrawMOID)
```

Comment:
The ifndef does matter. The spatial partition grid is populated by determining where the draws are registered, so g_DrawMOID is still used - it just doesn’t draw to the MOID layer, instead it triggers the spatial partition register. This isn’t nice, but it was the least intrusive way at the time. In my multithreaded sim branch, this is cleaned up a bit (registers once, instead of per pass), and later I intend to fully remove g_DrawMOID and instead do the spatial partition population in an altogether separate function.

Reply:
Yeah, makes sense. My bad really, I wrote this comment near the beginning of my review and I should have gone back through my comments to make sure I didn't wanna prune any. As you say, draw moid is still used (and I agree it's a hacky but reasonable solution to avoid rewriting even more stuff).